README.rst

```diff
@@ -5,7 +5,7 @@
 Travertino
 ==========
 
-.. image:: https://img.shields.io/pypi/pyversions/travertino.svg
```

Comment:
What's the reason for this change?

Reply:
I believe the original version shield was created using a selected one that is available on the shields.io website that lists versions 3.4, 3.5, 3.6. I had to create a new "static" shield that lists the updated versions we want to list (3.5, 3.6, 3.7).

Reply:
This isn't needed anymore; 3.7 is now officially supported. (see https://img.shields.io/pypi/pyversions/briefcase.svg)

Reply:
Got it, sorry I didn't realize these shields were dynamically generated